cmd/stolonctl/cmd/status.go

```diff
@@ -282,12 +284,28 @@ func generateStatus() (Status, error) {
 		return status, err
 	}
 
+	cluster := ClusterStatus{}
+	if cd.Cluster == nil || cd.DBs == nil {
+		cluster.Available = false
+	} else {
+		master := cd.Cluster.Status.Master
+		cluster.Available = true
+
+		if master != "" {
+			cluster.MasterDBUID = cd.DBs[master].UID
+			cluster.MasterKeeperUID = cd.Keepers[cd.DBs[master].Spec.KeeperUID].UID
+		}
+	}
+	status.Cluster = cluster
+
 	keepers := make([]KeeperStatus, 0)
 	kssKeys := cd.Keepers.SortedKeys()
+	keeperRoles := getKeeperRoles(status.Cluster.MasterDBUID, cd)
 	for _, kuid := range kssKeys {
 		k := cd.Keepers[kuid]
 		db := cd.FindDB(k)
 		dbListenAddress := "(no db assigned)"
+		role := ""
```

Comment:
Better to set it to something like `none` instead of an empty value

Reply:
@sgotti, would setting the none value to `no role assigned` be acceptable? The reason of proposing this is that I found it a bit confusing displaying (none) next to the keeper. This is how it looks like in both cases:

With `none`:
```
===== Keepers/DB tree =====

keeper0 (master)
  keeper1 (sync)
  keeper2 (none)
```

With `no role assigned`:
```
===== Keepers/DB tree =====

keeper0 (master)
  keeper1 (sync)
  keeper2 (none)
```

Which do you think works better?